samples/msal-node-samples/ElectronSystemBrowserTestApp/playwright.config.ts

```diff
@@ -0,0 +1,20 @@
+import { PlaywrightTestConfig, devices } from "@playwright/test";
+
+const config: PlaywrightTestConfig = {
+    testDir: "./tests",
+    maxFailures: 2,
+    use: {
+        headless: false,
+        trace: "on-first-retry",
+    },
+    projects: [
+        {
+            name: "chromium",
+            use: { ...devices["Desktop Chrome"] },
+        },
+    ],
+    timeout: 30000,
+    globalTimeout: 5400000,
```

Comment:
can we add `retries` to the config, using `RETRY_TIMES` constant from e2eTestUtils? (I think testUtils.ts)

Reply:
Done